src/assets/config/edition_config.json

```diff
@@ -109,7 +109,7 @@
   "notSignificantVariants": [
     "type=orthographic"
   ],
-  "defaultEdition": "diplomatic",
+  "defaultEdition": "critical",
```

Comment:
@RobertoRDT it's fine for you that the default edition level will be "critical" instead of "diplomatic"?